src/shapes/IText/IText.ts

```diff
@@ -18,6 +18,28 @@ type CursorBoundaries = {
   topOffset: number;
 };
 
+export const iTextDefaultValues = {
+  selectionStart: 0,
+  selectionEnd: 0,
+  selectionColor: 'rgba(17,119,255,0.3)',
+  isEditing: false,
+  editable: true,
+  editingBorderColor: 'rgba(102,153,255,0.25)',
+  cursorWidth: 2,
+  cursorColor: '',
+  cursorDelay: 1000,
+  cursorDuration: 600,
+  caching: true,
+  hiddenTextareaContainer: null,
+  _selectionDirection: null,
+  _reSpace: /\s|\n/,
+  inCompositionMode: false,
```

Comment:
these 3 are not really options.
`_selectionDirection` and `inCompositionMode` are state.
`_reSpace` is static?

Reply:
they are what they are before this pr, and this PR is just moving them away from the prototype.
I didn't even read the default values in this PR.
But yes those aren't defaults values and will be moved probably somewhere else